entity-framework/core/miscellaneous/connection-resiliency.md

```diff
@@ -10,10 +10,7 @@ uid: core/miscellaneous/connection-resiliency
 
 # Connection Resiliency
 
-> [!NOTE]  
-> This feature is new in EF Core 1.1.
-
-Connection resiliency automatically retries failed database commands. It is specific to relational databases.
+Connection resiliency automatically retries failed database commands. It is supported by all relational databases.
```

Comment:
Does "supported" here imply that it works out-of-the-box for any relational database?

Reply:
An execution strategy can be specified for any relational provider. But we only provide one for SQL Server

Reply:
Yes, I understand that, I just think "supported" is ambiguous here. @divega can make the final call.

Reply:
I would say something like connection resiliency _can_ be supported by any relational provider by implementing an execution strategy, and that the SQL Server provider does it.

Reply:
I ended up coming up with something a bit more elaborate and more appropriate for the context (given the next section is on the SQL Server execution strategy). I will send a separate PR for review.